kfs-core/src/main/resources/edu/arizona/kfs/coa/spring-coa.xml

```diff
@@ -1,25 +1,53 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I don't understand why you replaced this block of code that was already in place and working for another mod. All you needed to do was add the bean definitions that were required for your mod. Please reinstate this code. Your datadictionary packages paths are not correct.